app/models/custom_link.rb

```diff
@@ -0,0 +1,23 @@
+class CustomLink < ApplicationRecord
+  belongs_to :casa_org
+end
```

Comment:
Should add some validation on the url

Reply:
This task is still in progress... Will definitely add those